SDWebImage/SDWebImageWebPCoder.m

```diff
@@ -25,6 +25,30 @@
 #endif
 #import <Accelerate/Accelerate.h>
 
+// Create and return the correct colorspace by checking the ICC Profile
+static CGColorSpaceRef SDColorSpaceCreateWithDemuxer(WebPDemuxer *demuxer) {
```

Comment:
Emm...Does this really need using a C function instead of Objective-C method ? And my previous code is to used as a non-null method, however, this one become a nullable.

Reply:
> // Marks functions which return a CF type that needs to be released by the caller but whose names are not consistent with CoreFoundation naming rules. The recommended fix to this is to rename the functions, but this macro can be used to let the clang static analyzer know of any exceptions that cannot be fixed.

Emm, Objective-C or C are both ok, what I want to change actually is make function name satisfy `CoreFoundation` naming rules, it's what `Apple` recommend and I think it may more better expression~

Reply:
...I dislike this naming, actually. It should about the WebP, but not using a common `SDColorSpaceCreateWithDemuxer`. We can just directlly fix the potential leak issue.

Reply:
Emm, I like `C` for Core Foundation APIs, we can add some macros like Apple's Core Foundation API, the usage is the same IMO.

For this `PR`, the naming is very very trivial, let's fix the problem and do some optimize. I'll revert the name.